website/src/controllers/api-v3/chat.js

```diff
@@ -256,4 +256,40 @@ api.flagChat = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/chat/:chatId/seen Seen a group chat message
+ * @apiVersion 3.0.0
+ * @apiName SeenChat
+ * @apiGroup Chat
+ *
+ * @apiParam {groupId} groupId The group _id
+ *
+ * @apiSuccess {None}
+ */
+api.seenChat = {
+  method: 'POST',
+  url: '/groups/:groupId/chat/seen',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    let user = res.locals.user;
+    let groupId = req.params.groupId;
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId});
+    if (!group) throw new NotFound(res.t('groupNotFound'));
+
+    // Skip the auth step, we want this to be fast. If !found with uuid/token, then it just doesn't save
```

Comment:
the auth middleware is used (which is :+1: for me) so we should remove this comment probably